packages/@okta/vuepress-site/docs/guides/protect-your-api/main/aspnet/samplecode.md

```diff
@@ -1 +1,2 @@
-See the [ASP.NET Web API quickstart](https://docs.microsoft.com/en-us/aspnet/web-api/overview/getting-started-with-aspnet-web-api/tutorial-your-first-web-api) for the basics of building a Web API project to follow the guide with, or download a finished sample app from our [ASP.NET example repo](https://github.com/okta/samples-aspnet/tree/master/resource-server).
\ No newline at end of file
+* [ASP.NET Web API quickstart](https://docs.microsoft.com/en-us/aspnet/web-api/overview/getting-started-with-aspnet-web-api/tutorial-your-first-web-api) to build a Web API project to follow the guide with
```

Comment:
I just find this a bit clunky. Maybe ` .... to build a Web API project` alone or  `...to build a Web API project for this guide`

Reply:
👍